bindings/java/src/org/sleuthkit/datamodel/OsAccount.java

```diff
@@ -399,10 +399,11 @@ void resetDirty() {
 	
 	
 	/**
-	 * Adds account attributes to the account. Attributes can be at
+	 * Adds account attributes to the account.Attributes can be at
```

Comment:
I think the period is helpful here.

Reply:
I am not sure what your comment is refering too, but ss mentioned in a comment to @APriestman the header formatting you see on github is not matching the file.